translator/sample files/navjoy_data/workzone_construction_closure.json

```diff
@@ -0,0 +1,262 @@
+[
```

Comment:
I don't see a reference to this file, are we using?

Reply:
This is not being used, I left it here just in case we need to use it eventually (in case we don't like NavJoy's WZDx feed). Want me to delete it?

Reply:
Is this an example of their data feed they use to generate their own WZDx feed? Maybe as above, we can just note in the README what it is.

Reply:
same as above